src/data/documentation-links.yaml

```diff
@@ -46,36 +46,38 @@
       items:
         - id: Introduction
           to: /docs/interacting-with-geth/rpc
+        - id: RPC Namespaces
+          items:
+          - id: admin
+            to: /docs/interacting-with-geth/rpc/ns-admin
+          - id: clique
+            to: /docs/interacting-with-geth/rpc/ns-clique
+          - id: debug
+            to: /docs/interacting-with-geth/rpc/ns-debug
+          - id: eth
+            to: /docs/interacting-with-geth/rpc/ns-eth
+          - id: les
+            to: /docs/interacting-with-geth/rpc/ns-les
+          - id: miner
+            to: /docs/interacting-with-geth/rpc/ns-miner
+          - id: net
+            to: /docs/interacting-with-geth/rpc/ns-net
+          - id: Personal (deprecation notes)
+            to: /docs/interacting-with-geth/rpc/ns-personal-deprecation
+          - id: Personal (deprecated)
```

Comment:
Why are there two pages related to `personal`? I think we could have the reason for deprecation at the top of the reference docs page.

Reply:
yea we could - that just creates a very long page. The deprecation notes include mapping out the deprecated methods to their replacements.

Your call - if you prefer a very long single page then no problem

Reply:
I think it'd be better to have only a single page. I don't think it's a problem to have a long page, it's more important to have a 'canonical' page for the personal namespace. Long term, we will remove this page anyway.